src/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface Label {
```

Comment:
Don't know why you removed `I` (`ILabel`), by the typescript and eslint rules interfaces are called `I` - interface `Label` - interface  name.

Reply:
Cant see any value of prefexing with `I`, my last commit removed it. 
* TypeScript itself of against it https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names
* vue.js do not use them
* web-ui mixed :)